src/Resiliency/BackoffStrategies/Jitter/DecorrelatedJitterBackoffStrategy.cs

```diff
@@ -7,34 +8,39 @@ namespace Resiliency.BackoffStrategies.Jitter
     /// <para/>
     /// Ranges from the wait time given by the underlying strategy up to 3 times the last given wait time.
     /// </summary>
-    public class DecorrelatedJitterBackoffStrategy : IBackoffStrategy
+    public class DecorrelatedJitterBackoffStrategy 
+        : IBackoffStrategy
     {
         private readonly IBackoffStrategy _strategy;
-        private readonly TimeSpan _lastWaitTime;
         private readonly IRandomNumberGenerator _randomNumberGenerator;
 
-        public DecorrelatedJitterBackoffStrategy(IBackoffStrategy strategy, TimeSpan lastWaitTime, IRandomNumberGenerator randomNumberGenerator)
+        private TimeSpan _lastWaitTime;
+
+        public DecorrelatedJitterBackoffStrategy(IBackoffStrategy strategy)
+            : this(strategy, new DefaultRandomNumberGenerator())
         {
-            if (lastWaitTime < TimeSpan.Zero)
-                throw new ArgumentException("The last wait time cannot be less than zero.", nameof(lastWaitTime));
+        }
 
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public DecorrelatedJitterBackoffStrategy(IBackoffStrategy strategy, IRandomNumberGenerator randomNumberGenerator)
+        {
             _strategy = strategy ?? throw new ArgumentNullException(nameof(strategy));
-            _lastWaitTime = lastWaitTime;
+            _lastWaitTime = _strategy.InitialWaitTime;
             _randomNumberGenerator = randomNumberGenerator ?? throw new ArgumentNullException(nameof(randomNumberGenerator));
         }
 
         TimeSpan IBackoffStrategy.InitialWaitTime => _strategy.InitialWaitTime;
 
-        public TimeSpan GetWaitTime(int attemptNumber)
+        public TimeSpan Next()
         {
-            if (attemptNumber < 1)
-                throw new ArgumentException($"The number of attempts cannot be less than 1 when getting the wait time of an {nameof(IBackoffStrategy)}.", nameof(attemptNumber));
+            var waitTime = TimeSpan.FromMilliseconds(
+                _randomNumberGenerator.Next(
+                    minValue: _strategy.Next().TotalMilliseconds, 
+                    maxValue: _lastWaitTime.TotalMilliseconds * 3));
 
-            var waitTimeMs = 
-                _randomNumberGenerator.NextDouble() * (_lastWaitTime.TotalMilliseconds * 3 - _strategy.InitialWaitTime.TotalMilliseconds) 
-                + _strategy.InitialWaitTime.TotalMilliseconds;
+            _lastWaitTime = waitTime;
```

Comment:
I'm still undecided on adding state (nothing against it yet, though), but this looks much cleaner.